privacyidea/lib/eventhandler/base.py

```diff
@@ -333,6 +339,18 @@ def check_condition(self, options):
                               "fulfilled.".format(token_obj))
                     res = False
 
+            if CONDITION.TOKEN_IS_EXPIRED in conditions and res:
+                expired = not token_obj.check_validity_period()
+                check = conditions.get(CONDITION.TOKEN_IS_EXPIRED)
+                if expired and check in ["True", True]:
+                    res = True
+                elif not expired and check in ["False", False]:
+                    res = True
+                else:
+                    log.debug("Condition token_is_expired for token {0!r} not "
+                              "fulfilled.".format(token_obj))
+                    res = False
```

Comment:
This will also increase the code coverage. At the moment the elsid and else are not covered ;-)
Hm, you could also add a negative test to cover the log output.